templates/articles/detail.html

```diff
@@ -2,22 +2,22 @@
 <div class="article-page">
   <div class="banner">
     <div class="container">
-      <h1>How to build webapps that scale</h1>
+      <h1>{{ article.title }}</h1>
 
       <div class="article-meta">
-        <a href=""><img src="http://i.imgur.com/Qr71crq.jpg" /></a>
+        <a href=""><img src="{{ article.author.img }}" /></a>
         <div class="info">
-          <a href="" class="author">Eric Simons</a>
-          <span class="date">January 20th</span>
+          <a href="" class="author">{{ article.author }}</a>
```

Comment:
One more missing link:
```suggestion
          <a href="{% url 'profile_view' article.author.user.username %}" class="author">{{ article.author }}</a>
```